src/Pickles/Pickles.DocumentationBuilders.Cucumber/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System.Reflection;
```

Comment:
Please use a minimal AssemblyInfo.cs similar to the other projects, and link the VersionInfo.cs file to the project.